dumper/dataCrawler.go

```diff
@@ -140,18 +142,20 @@ func shouldFollowReferenceToLink(path []string, currentTable schemareader.Table,
 		}
 	}
 
-	forcedNavegations := map[string] []string {
+	forcedNavegations := map[string][]string{
 		"rhnchannelfamily": {"rhnpublicchannelfamily"},
-		"rhnchannel": {"susemddata", "suseproductchannel", "rhnreleasechannelmap", "rhndistchannelmap"},
-		"suseproducts": {"suseproductextension", "suseproductsccrepository"},
-		"rhnpackageevr": {"rhnpackagenevra"},
-		"rhnerrata": {"rhnerratafile"},
-		"rhnerratafile": {"rhnerratafilechannel"},
+		"rhnchannel":       {"susemddata", "suseproductchannel", "rhnreleasechannelmap", "rhndistchannelmap"},
+		"suseproducts":     {"suseproductextension", "suseproductsccrepository"},
+		"rhnpackageevr":    {"rhnpackagenevra"},
+		"rhnerrata":        {"rhnerratafile"},
+		"rhnerratafile":    {"rhnerratafilechannel"},
+		"rhnconfigchannel": {"rhnconfigfile"},
+		"rhnconfigfile":    {"rhnconfigrevision"},
 	}
 
 	if tableNavigation, ok := forcedNavegations[currentTable.Name]; ok {
-		for _, targetNavigationTable := range tableNavigation{
-			if strings.Compare(targetNavigationTable, referencedTable.Name) == 0{
+		for _, targetNavigationTable := range tableNavigation {
+			if strings.Compare(targetNavigationTable, referencedTable.Name) == 0 {
 				return true
 			}
 		}
```

Comment:
Unit tests in [crawler_test](https://github.com/uyuni-project/inter-server-sync/pull/22/files#diff-694c17d81c9f4858d537e991fc26450c36faf50fe07b5caa1176c2f006534f9d)